packages/devtools/src/sourceMap/loadOriginalSourceLoc.ts

```diff
@@ -0,0 +1,178 @@
+        //   console.groupEnd();
+        // }
+
+        if (sourceMapIncludesSource(sourceMapJSON, getFilePath(source))) {
+          // store computed sourceMapJSON
+          if (sourceMapNum > 1) {
+            sourceMapJSONs.set(`${source}:${line}:${column}`, sourceMapJSON);
+          } else {
+            if (sourceMapRegex.exec(runtimeSourceCode) === null) {
+              sourceMapJSONs.set(source, sourceMapJSON);
+            } else {
+              sourceMapJSONs.set(`${source}:${line}:${column}`, sourceMapJSON);
+            }
+          }
+
+          return getOriginalPosition(sourceMapJSON, sourceLoc);
+        }
+      } catch (error) {
+        // We've likely encountered a string in the source code that looks like a source map but isn't.
+        // Maybe the source code contains a "sourceMappingURL" comment or something similar.
+        // In either case, let's skip this and keep looking.
+      }
+    } else {
+      externalSourceMapURLs.push(sourceMappingURL);
+    }
+
+    sourceMappingURLMatch = sourceMapRegex.exec(runtimeSourceCode);
+  }
+
+  // No inline sourcemap found. Check if there's external source map
+  let sourceMappingURL = externalSourceMapURLs.pop();
+  if (!sourceMappingURL) {
+    throw new Error(
+      `[Griffel devtools] extractAndLoadSourceMapJSON() encountered string "sourceMappingURL" in file ${source} that does not contain a valid source map`,
+    );
+  }
+
+  if (externalSourceMapURLs.length > 0) {
+    console.warn(
+      `[Griffel devtools] extractAndLoadSourceMapJSON() More than one external source map detected in the file ${source}:`,
+    );
+    externalSourceMapURLs.forEach(sourceMappingURL => {
+      console.warn(`[Griffel devtools] extractAndLoadSourceMapJSON() skipping source map "${sourceMappingURL}"`);
+    });
+  }
+
+  if (!sourceMappingURL.startsWith('http') && !sourceMappingURL.startsWith('/')) {
+    const lastSlashIdx = source.lastIndexOf('/');
+    if (lastSlashIdx !== -1) {
+      const baseURL = source.slice(0, lastSlashIdx);
+      sourceMappingURL = `${baseURL}/${sourceMappingURL}`;
+    }
+  }
+
+  const sourceMapContent = await fetchFiles(sourceMappingURL);
+  const sourceMapJSON = JSON.parse(sourceMapContent);
+  sourceMapJSONs.set(source, sourceMapJSON);
+
+  return getOriginalPosition(sourceMapJSON, sourceLoc);
+}
+
+async function fetchFiles(url: string): Promise<string> {
+  try {
+    const response = await fetch(url);
+    if (response.ok) {
+      return response.text();
+    }
+    throw new Error(`[Griffel devtools] fetchRuntimeSource() bad response fetching ${url}: ${response.status}`);
+  } catch (error) {
+    // try to get url content from chrome resources api
+    const resource = (await resources).find(resource => resource.url === url);
+    if (resource) {
+      return new Promise(resolve => resource.getContent(content => resolve(content)));
+    }
+    throw new Error(`[Griffel devtools] fetchFiles() unable to fetch ${url}: ${error}`);
```

Comment:
```suggestion
```

You never need this error right ? you just want to rethrow the earlier error. If the chrome resource api fetch fails, it will throw a compeltely different error

Reply:
Hmm I didn't get it? I think I need it: if fetch fails, it will be catched, and then I will re-throw it here

Reply:
I made a type in the original suggestion, you should just rethrow the caught error:

If original fetch throws an error and you try to use `resource.getContent`

If `resource.getContent` throws then a different error is thrown.

So there is no reason to rethrow a new error from the `catch` block

Reply:
Oh gotcha!